api/views.py

```diff
@@ -29,4 +29,18 @@ def post(self, request):
                     'correct_answer': cached_response['answer']}
         except Exception:
             return Response("Must supply answer and key", status=status.HTTP_400_BAD_REQUEST)
+        return Response(data)
+
+class HelpSteps(LoginRequiredMixin, APIView):
+
+    def post(self, request):
+        try:
+            question_key = request.data['key']
+            cached_response = cache.get(question_key)
+            data = {'total_package': cached_response}
```

Comment:
We ONLY need the help steps for this route.  The question and answer routes already contain the rest of the info.

Reply:
Okay cool, just made some changes